fe/fe-core/src/main/java/com/starrocks/lake/compaction/PartitionStatistics.java

```diff
@@ -83,8 +102,23 @@ public Quantiles getCompactionScore() {
         return compactionScore;
     }
 
+    public CompactionPriority getPriority() {
+        // For backward compatibility
+        // prevent null value when deserializing JSON that doesn't include the priority field
+        return priority == null ? CompactionPriority.DEFAULT : priority;
+    }
+
+    public synchronized void setPriority(CompactionPriority priority) {
```

Comment:
```suggestion
    public void setPriority(CompactionPriority priority) {
```
Since you've defined `priority` as `volatile` variable, there is no need to define the method as `synchronized`?

Reply:
> Since you've defined `priority` as `volatile` variable, there is no need to define the method as `synchronized`?

yes, i will remove it